samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java

```diff
@@ -138,6 +146,14 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
 		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
 	}
 
+	@After("@requireMFAGroupsForUser")
```

Comment:
If I understand what this is doing, it's removing the user from the everyone group. Does that work?

In standups, I think we discussed having 2 different facebook users, so we don't have to do this.

Reply:
Yes, this is the first solution that made before the discussion. Vijet already created a separate facebook user with MFA and I'll fix this.